packages/ua-devtools-evm-hardhat/src/tasks/oapp/config.init.ts

```diff
@@ -6,29 +6,33 @@ import { printLogo } from '@layerzerolabs/io-devtools/swag'
 import { createLogger, pluralizeNoun, printJson, setDefaultLogLevel } from '@layerzerolabs/io-devtools'
 import { getEidsByNetworkName, types } from '@layerzerolabs/devtools-evm-hardhat'
 import { promptToSelectMultiple } from '@layerzerolabs/io-devtools'
-import { OAppOmniGraphHardhat } from '@/oapp'
+import {
+    createPrinter,
+    createSourceFile,
+    ListFormat,
+    NodeArray,
+    ScriptKind,
+    ScriptTarget,
+    SourceFile,
+    Statement,
+} from 'typescript'
+import { generateLzConfig } from '@/oapp/typescript/typescript'
+import { writeFileSync } from 'fs'
 
 interface TaskArgs {
+    contractName: string
     oappConfig: string
     logLevel?: string
 }
 
-interface ConfigFile {
-    path: string
-    value: OAppOmniGraphHardhat
-}
-
 /**
  * This task will initialize a config file based on user input
- * and return an array of `ConfigFile` objects.
  *
- * The return value is an array to support cases where the users would select
- * a combination of mainnet/testnet networks in which case we'll split them into
- * multiple configurations per network stage.
+ * The return value is a file path to the newly generated LayerZero Config file
  *
- * @returns {Promise<ConfigFile[]>}
+ * @returns {Promise<oappConfig>}
```

Comment:
Should just be `Promise<string>`, as `oappConfig` is not a type.  Alternatively, declare a type for it similar to:

```
type OAppConfigFilePath = string
```